GameOfLife/GameOfLife/Views/Render.cs

```diff
@@ -0,0 +1,312 @@
+                "\n Every cell interacts with its eight neighbours, which are the cells that are horizontally, vertically, or diagonally adjacent. " +
+                "\n At each step in time, the following transitions occur:");
+            Console.WriteLine("\n # Any live cell with fewer than two live neighbours dies, as if by underpopulation.");
+            Console.WriteLine(" # Any live cell with two or three live neighbours lives on to the next generation.");
+            Console.WriteLine(" # Any live cell with more than three live neighbours dies, as if by overpopulation.");
+            Console.WriteLine(" # Any dead cell with exactly three live neighbours becomes a live cell, as if by reproduction.");
+            Console.WriteLine("\n These rules, which compare the behavior of the automaton to real life, can be condensed into the following:");
+            Console.WriteLine("\n Any live cell with two or three live neighbours survives.");
+            Console.WriteLine(" # Any dead cell with three live neighbours becomes a live cell.");
+            Console.WriteLine(" # All other live cells die in the next generation.Similarly, all other dead cells stay dead.");
+            Console.WriteLine("\n The initial pattern constitutes the seed of the system." +
+                "\n The first generation is created by applying the above rules simultaneously to every cell in the seed, live or dead; " +
+                "\n births and deaths occur simultaneously, and the discrete moment at which this happens is sometimes called a tick. " +
+                "\n Each generation is a pure function of the preceding one.The rules continue to be applied repeatedly to create further generations.");
+            Console.WriteLine("\n # Press any key to go back");
+            Console.ReadKey();
+        }
+
+        /// <summary>
+        /// Method for rendering the UI when all the cells on the field are dead.
+        /// </summary>
+        /// <param name="generation">Parameter that represents the generation number.</param>
+        public void GameOverRender(int generation)
+        {
+            Console.Clear();
+            for (int i = 0; i < 5; i++)
+            {
+                Console.WriteLine(DashesConstant);
+            }
+            Console.WriteLine(FieldDeadPhrase);
+            Console.WriteLine($"\n Generations survived: {generation}");
+        }
+
+        /// <summary>
+        /// Method for rendering the UI for choosing which saved game file to load.
+        /// </summary>
+        /// <param name="numberOfFiles">The number of saved game files currently present in the folder.</param>
+        public void ChooseFileToLoadMenuRender(int numberOfFiles, string filePath, bool wrongInput)
+        {
+            Console.CursorVisible = true;
+            Console.Clear();
+            Console.WriteLine(" ### Choose which saved game to load ###");
```

Comment:
Code should not contain so called 'magic strings' or 'magic numbers'.
Text or number literals should be placed into resource file or in constant variable at the start of class.

Reply:
Wil discuss this personally